apps-rendering/src/components/Callout/shareLink.tsx

```diff
@@ -0,0 +1,44 @@
+import type { ArticleFormat } from '@guardian/libs';
+import { Button, SvgShareCallout } from '@guardian/source-react-components';
+import { useState } from 'react';
+import type { FC } from 'react';
+import { calloutShare, calloutSharelink } from './styles';
+
+export const ShareLink: FC<{ format: ArticleFormat }> = ({ format }) => {
+	const [isCopied, setIsCopied] = useState(false);
```

Comment:
I'm not sure if we need this state? From what I can tell it gets set to true then set to false in line 22/23 but isn't being used for anything  - can we get rid of it?

Reply:
Its used to determine whether the `Link copied to clipboard` text is rendered -[ line 41](https://github.com/guardian/dotcom-rendering/pull/6800/files#diff-7e7918a3b84020e0100f847598aa1ae7273ec41877d4156b63293921bc35a96eR41)

Reply:
of course it is! - coffee hadn't hit yet